tools/configgen/pkg/testgrid.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sort"
 	"strings"
 
-	"knative.dev/test-infra/pkg/testgrid"
+	"knative.dev/infra/pkg/testgrid"
```

Comment:
if we can point this at toolbox then we can remove it from this repo

Reply:
Not quite, I tried it and this happens:

```
# istio.io/test-infra/tools/prowgen/pkg
../../../../../go/pkg/mod/istio.io/test-infra/tools/prowgen@v0.0.0-20220912223856-cd655368c7d2/pkg/generate.go:257:30: undefined: client.GerritReportLabel
../../../../../go/pkg/mod/istio.io/test-infra/tools/prowgen@v0.0.0-20220912223856-cd655368c7d2/pkg/generate.go:309:31: undefined: client.GerritReportLabel
```

I'll investigate this in a separate PR.

Reply:
I saw that error when i was blowing away the vendor. So, might just be a vendoring thing. However, it is fine we punt this until we can relocate configgen to toolbox